download_pdfs.py

```diff
@@ -1,53 +1,50 @@
-import os
+import os, yaml
+
 from playwright.sync_api import Playwright, sync_playwright
-def run(playwright: Playwright) -> None:
-    # Read list of tems dashboards and add tehm to dashboards
-    with open('teams_dashboards.txt') as data_file:
-        dashboards = data_file.readlines()
 
-    # Specify the download directory
-    download_path = "downloaded_pdfs/"
 
-    browser = playwright.chromium.launch(headless=False
-    )
+def run(playwright: Playwright) -> None:
+    #Read and load weekly_report.yaml file
+    with open("weekly_report.yaml", "r")as yaml_file:
+        teams_data = yaml.safe_load(yaml_file)
```

Comment:
The code comment at line 7 no longer says anything which the code itself doesn't clearly say.    You should either delete it or enhance it.  (And, if you keep it, both Flake8 and Black will insist on a space after the `#`.  🙂)

(Similar comments apply to lines 11 and 18.)